schema/postgresql.sql

```diff
@@ -524,34 +524,34 @@ BEGIN
 END;
 $$ LANGUAGE plpgsql;
 
--- Function to authenticate a user with username and password
-CREATE OR REPLACE FUNCTION get_user_for_login(p_username VARCHAR(64))
-RETURNS TABLE (user_id VARCHAR(64), username VARCHAR(64), password_hash TEXT) AS $$
+-- Function to authenticate a user with email and password
+CREATE OR REPLACE FUNCTION get_user_for_login(p_email VARCHAR(64))
```

Comment:
Does this need to be updated, too?
```suggestion
CREATE OR REPLACE FUNCTION get_user_for_login(p_email VARCHAR(255))
```